panel/widgets/chat.py

```diff
@@ -0,0 +1,1646 @@
+"""The chat module provides components for building and using chat interfaces
+
```

Comment:
The "copy button* is really useful. Two things would improve it

- That something visible happens when you click it. When nothing happens you are not sure it has copied the text.
- The the "copy button" was located to the right (together with the heart) like copy buttons normally are.

![image](https://github.com/holoviz/panel/assets/42288570/e50c1e90-b678-4785-bb53-8999ffeb7ceb)

Reply:
I prefer the copy button on the left to prevent confusion from reactions.